net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml

```diff
@@ -42,6 +42,10 @@
                                 <Required>Y</Required>
                                 <mask>/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/</mask>
                         </area>
+                        <passive type="BooleanField">
+                                <default>0</default>
+                                <Required>N</Required>
```

Comment:
should be required - will be defaulted anyway especially when you expect it to be there.

Reply:
Did you change the wrong property?

Reply:
Whoopsie .. sorry